Generated/Ontology.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
I'm working on using the same .nuspec for the OAS API spec generation tooling, where much of the same metadata (id, description, authors, version, etc) are used.
Here's that file: 
https://raw.githubusercontent.com/RealEstateCore/rec/feature-REST-API-REC4/Ontology/RealEstateCore.Ontology.DTDLv2.nuspec

It'd be great if we could align on the naming and contents. I'm thinking the term "Ontology" in singular (since there's only one, consisting of many interfaces), and including the DTDL version in the .nuspec file name (since there'll be one Nuget package generated per DTDL version, so I assume multiple .nuspec files). Is that OK?